<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalculatorWindow</class>
 <widget class="QDialog" name="CreditCalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit calculator</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(25, 25, 25);</string>
   </property>
   <widget class="QLineEdit" name="termLineEdit">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>80</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244);
background-color: rgb(36, 31, 49);
border-style: solid;
border-color: rgb(118, 130, 144);
	border-width: 1px;
    border-radius: 0px;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonAnnuity">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>160</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244);
</string>
    </property>
    <property name="text">
     <string>Annuity</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="interestRateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244)</string>
    </property>
    <property name="text">
     <string>Interest rate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="creditTypeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244)</string>
    </property>
    <property name="text">
     <string>Credit type:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDifferentiated">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244)</string>
    </property>
    <property name="text">
     <string>Differentiated</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="creditAmountLineEdit">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244);
background-color: rgb(36, 31, 49);
border-style: solid;
border-color: rgb(118, 130, 144);
	border-width: 1px;
    border-radius: 0px;</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="creditAmountLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244)</string>
    </property>
    <property name="text">
     <string>Credit amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="termLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244)</string>
    </property>
    <property name="text">
     <string>Term:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="interestRateLineEdit">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>120</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244);
background-color: rgb(36, 31, 49);
border-style: solid;
border-color: rgb(118, 130, 144);
	border-width: 1px;
    border-radius: 0px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculatePushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: rgb(48, 53, 70);
    color: rgb(228, 226, 230);
    font-weight: bold;
border-style: solid;
border-color: rgb(118, 130, 144);
	border-width: 1px;
    border-radius: 0px;</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="calculationResultLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244);
background-color: rgb(36, 31, 49);
border-style: solid;
border-color: rgb(118, 130, 144);
	border-width: 1px;
    border-radius: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="termTimeTypeComboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 31, 49);
color: rgb(246, 245, 244);
border-style: solid;
border-color: rgb(118, 130, 144);
	border-width: 1px;
    border-radius: 0px;</string>
    </property>
    <item>
     <property name="text">
      <string>years</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>months</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244);</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 245, 244);
</string>
    </property>
    <property name="text">
     <string>â‚½</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>creditAmountLineEdit</tabstop>
  <tabstop>termLineEdit</tabstop>
  <tabstop>interestRateLineEdit</tabstop>
  <tabstop>termTimeTypeComboBox</tabstop>
  <tabstop>radioButtonAnnuity</tabstop>
  <tabstop>radioButtonDifferentiated</tabstop>
  <tabstop>calculatePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
